components/i2c/sen66/definition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "displayName": "SEN66",
+  "vendor": "Sensirion",
+  "productURL": "https://www.digikey.ch/en/products/detail/sensirion-ag/SEN66-SIN-T/25700945?s=N4IgTCBcDaIM4FMB2A2FIC6BfIA",
```

Comment:
If you wouldn't mind changing this to be .com instead of .ch and dropping the query string / attribution bit too (unless your keen to keep it)